components/sync_manager/README.md

```diff
@@ -0,0 +1,13 @@
+# Sync Manager
+
+It's a bit unfortunate this component can't just be part of `sync15`.
+Conceptually and functionally, it shares most in common with with the `sync15`
+crate, and in some cases stretches (or arguably breaks) the abstraction barrier
+that `sync15` puts up.
```

Comment:
I was thinking the exact same thing working on the clients engine. 😅 We may want to rethink Places and Logins depending on `sync15` (or keeping shared types for the syncable store there, and moving the guts of the syncing logic into sync manager...so, say, Places wouldn't have `sync_history` or `sync_bookmarks` methods anymore), but this isn't the time or place to rearchitect that. Thanks for adding the note to the readme!